app/Kconfig

```diff
@@ -1,5 +1,5 @@
-# Copyright (c) 2020 The ZMK Contributors
-# SPDX-License-Identifier: MIT
+#Copyright(c) 2020 The ZMK Contributors
```

Comment:
Were all the whitespace changes here intentional?

Reply:
Nope! Reverted.